src/app/cluster/details/external-cluster/template.html

```diff
@@ -35,7 +35,8 @@
         <span>Disconnect</span>
       </button>
 
-      <button id="km-delete-external-cluster-btn"
+      <button *ngIf="cluster?.labels['isImported'] === 'false'"
```

Comment:
What if there is no `isImported` label or no labels at all? I think this check should be like this:

```suggestion
      <button *ngIf="!cluster?.labels || cluster.labels['isImported'] !== 'true'"
```